--- ./vars.jsonnet
+++ ./vars.jsonnet
@@ -12,7 +12,7 @@
     },
     {
       name: 'armExporter',
-      enabled: false,
+      enabled: true,
       file: import 'modules/arm_exporter.jsonnet',
     },
     {
@@ -49,11 +49,11 @@

   k3s: {
     enabled: false,
-    master_ip: ['192.168.1.15'],
+    master_ip: ['192.168.86.200'],
   },

   // Domain suffix for the ingresses
-  suffixDomain: '192.168.1.15.nip.io',
+  suffixDomain: 'lan',
   // Additional domain suffixes for the ingresses.
   // For example suffixDomain could be an external one and this a local domain.
   additionalDomains: [],
